customer-segment-template-extension/src/CustomerSegmentTemplate.liquid

```diff
@@ -10,13 +10,17 @@ const TARGET = 'admin.customers.segmentation-templates.render';
 export default reactExtension(TARGET, () => <App />);
 
 function App() {
+  // The useApi hook provides access to several useful APIs like i18n, close, and data.
+  const { i18n } = useApi(TARGET);
+
   const query = 'number_of_orders = 1 AND products_purchased(id: (product_id)) = true';
   const queryToInsert = 'number_of_orders = 1 AND products_purchased(id: (';
 
   return (
     // The CustomerSegmentTemplate component provides an API for setting the title, description, date, query, and query to insert of the segmentation template.
-    // The title and description do not need to be explicitly set as they will be obtained from the extension's configuration file (TOML).
     <CustomerSegmentTemplate
+        title={i18n.translate("title")}
+        description={i18n.translate("description)}
```

Comment:
```suggestion
        description={i18n.translate("description")}
```

Reply:
One more note on single vs double quote. Nothing breaking it's just the date uses a single one.